src/main/java/no/sikt/nva/pubchannels/handler/FetchJournalRequest.java

```diff
@@ -0,0 +1,76 @@
+package no.sikt.nva.pubchannels.handler;
+
+import static nva.commons.core.attempt.Try.attempt;
+import java.time.LocalDate;
+import java.util.UUID;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.StringUtils;
+
+public final class FetchJournalRequest {
```

Comment:
Is this a validator class? The class name does not match what it does?

Reply:
It is a class representing data we use from a fetch journal request. Internally, it makes sure it has valid data. I agree that its purpose is not clear. I will try to improve it.